src/xopen/__init__.py

```diff
@@ -269,6 +277,10 @@ def __init__(
         mode: str = "r",
         threads_flag: Optional[str] = None,
         threads: Optional[int] = None,
+        *,
+        encoding=None,
```

Comment:
Should the default perhaps be `"utf-8"`? See:
https://docs.python.org/3.11/library/io.html#opt-in-encodingwarning
> If you are providing an API that uses open() or TextIOWrapper and passes encoding=None as a parameter, you can use text_encoding() so that callers of the API will emit an EncodingWarning if they don’t pass an encoding. However, please consider using UTF-8 by default (i.e. encoding="utf-8") for new APIs.

That should be mostly backwards compatible because we haven’t supported Windows so far, and the default on most Unix systems should be UTF-8 already.

If we leave it at `None`, we should use the `text_encoding()` function as suggested.

Reply:
`io.text_encoding()` Is a 3.10 only feature. I will set it to UTF-8.